x/docs/client/cli/tx.go

```diff
@@ -88,3 +89,41 @@ func GetCmdShareDocument(cdc *codec.Codec) *cobra.Command {
 
 	return cmd
 }
+
+func GetCmdShareDocumentReceipt(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "share-receipt [recipient] [tx-hash] [document-uuid] [proof]",
```

Comment:
Can we rename it `send-receipt`. I think it could be grammatically more correct although it doesn't change much at all